package.json

```diff
@@ -25,7 +25,14 @@
     "extends": "terra"
   },
   "stylelint": {
-    "extends": "stylelint-config-terra"
+    "extends": "stylelint-config-terra",
```

Comment:
ah, i will revert this.

Reply:
actually...I think we should leave this be until we release our stylelint config with these changes